### PR TITLE
Fix: Add missing test coverage

### LINKED ISSUE #76: Fix test coverage for Points Preservation
**Description:**
Critical functionality (points preservation during attendance changes) lacks proper test coverage, allowing the points reset bug to go undetected.

**Missing Tests:**
- Points preservation in `MarkCommandTest`
- Points preservation in `UnmarkCommandTest`
- Integration tests combining points and attendance operations
- Boundary testing for point overflow scenarios

### PR DESCRIPTION
**Pull request by javierchuaby**
### Add comprehensive test coverage for points preservation

⚠️ **TODO after PR #92 merges**

**Changes:**
- ✅ Test points preservation in MarkCommand and UnmarkCommand
- ✅ Test integer overflow scenarios  
- ✅ Boundary testing for edge cases

Resolves #76

### CODE REVIEW

File: src/test/java/seedu/address/logic/commands/MarkCommandTest.java [hunk 1]
```diff
@@ -18,6 +18,7 @@
 import seedu.address.model.ModelManager;
 import seedu.address.model.UserPrefs;
 import seedu.address.model.person.Person;
+import seedu.address.testutil.PersonBuilder;
```
Comment (seanowww) [inline src/test/java/seedu/address/logic/commands/MarkCommandTest.java]: I like that you used Java Builders
